ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/compliance.scanConfigs.utils.tsx

```diff
@@ -95,6 +205,30 @@ export function convertFormikToScanConfig(formikValues: ScanConfigFormValues) {
     };
 }
 
+export function convertScanConfigToFormik(
```

Comment:
I had this stashed away when I had originally worked on the conversion. I think this should work:

```typescript
export function convertScanConfigStatusToFormik(
    scanConfigStatus: ComplianceScanConfigurationStatus
): ScanConfigFormValues {
    const { scanName, scanConfig } = scanConfigStatus;
    const clusters = scanConfigStatus.clusterStatus.map((status) => status.clusterId);

    let intervalType: 'DAILY' | 'WEEKLY' | 'MONTHLY' | null = null;
    let daysOfWeek: DayOfWeek[] = [];
    let daysOfMonth: DayOfMonth[] = [];

    let time = `${scanConfig.scanSchedule.hour.toString().padStart(2, '0')}:${scanConfig.scanSchedule.minute.toString().padStart(2, '0')}`;
    if (scanConfig.scanSchedule.hour >= 12) {
        time += ' PM';
    } else {
        time += ' AM';
    }

    switch (scanConfig.scanSchedule.intervalType) {
        case 'WEEKLY':
            intervalType = 'WEEKLY';
            daysOfWeek = scanConfig.scanSchedule.daysOfWeek.days.map(String) as DayOfWeek[];
            break;
        case 'MONTHLY':
            intervalType = 'MONTHLY';
            daysOfMonth = scanConfig.scanSchedule.daysOfMonth.days.map(String) as DayOfMonth[];
            break;
        case 'DAILY':
            intervalType = 'DAILY';
            break;
        case 'UNSET':
            intervalType = null;
            break;
        default:
            break;
    }

    return {
        parameters: {
            name: scanName,
            description: scanConfig.description ?? '',
            intervalType,
            time,
            daysOfWeek,
            daysOfMonth,
        },
        clusters,
        profiles: scanConfig.profiles,
    };
}
```